tests/ensure_provider_tests.py

```diff
@@ -11,14 +11,14 @@
 
 
 GET_NM_VERSION = """
-    - block:
+    - block:  # noqa name
```

Comment:
I guess you could easily add a name to the block and it would also help.

Reply:
The changes to the script and all changes that result from running it should be its own commit, too (unless it fites already in another category) to avoid a review of dozens of generated changes.

Reply:
Basically, change `tests/ensure_provider_tests.py` so that it produces code that is ansible-lint clean, rather than having it generate code with ansible-lint exceptions - this applies to this `noqa name` and any other exceptions.

Reply:
yes, and move this and related changes to their own commit, so they can independently be reviewed. Reviewing the changes for the script and the resulting generated code together with small changes might result in small, unrelated changes to be missed.